src/server/services/focus/focus-types.ts

```diff
@@ -112,7 +113,7 @@ export type DecisionFormatted =
   | 'buitenbehandeling';
 
 export type TextPartContents = (data: any, customData?: any) => string;
-export type LinkContents = (data: any, customData?: any) => Partial<LinkProps>;
```

Comment:
Why did you remove this?

Reply:
I changed it into a none partial since LinkProps already has optional parameters. So I thought it would be ok to change it in just LinkProps